articles/overview/what-is-quantum-computing.md

```diff
@@ -0,0 +1,54 @@
+---
+title: What is quantum computing?
+description: Learn what quantum computing is, and what a quantum computer can do
+author: natke
+ms.author: nakersha
+ms.date: 10/14/2019
+ms.topic: article
+uid: microsoft.quantum.overview.what
+---
+
+# What is quantum computing?
+
+Quantum computing is the application of quantum physics to the field of computer science.
+
+The theory of quantum physics posits that matter, at a quantum level is simultaneously in multiple states. And those many states interfere with each other like waves in a tide pool.
+
+Quantum computing stores information in quantum states of matter and uses quantum operations to compute on that information, by harnessing and learning to program quantum interference.
+
+Quantum computing might sound daunting, but with the right resources you can start building quantum applications today.
+
+## The qubit
+
+In quantum computing, a quantum bit is a unit of quantum information, like a classical bit. Where classical bits hold a single binary value such as a 0 or 1, a qubit can hold both values at the same time. This is called **superposition**.
+
+The act of **measurement** changes a qubit. It goes from being in the infinite states of superposition to one of the binary states of 0 or 1. Again, measurement of a qubit is unlike a classical bit, which when 0 is measured as 0, and when 1 is measured as 1.
```

Comment:
I'll leave the actual phrasing up to y'all, but figured I'd try to clarify what Eduardo's getting at for you @natke . Basically, in a single qubit, the superposition will always be just between |0> and |1>. _However_, the superposition can be arbitrarily weighted between the two (think of a unit vector in the xy plane pointing in any direction between -1 and 1) so that's where the "infinitely many" possible states comes from. 

(Just to cover my keister in case Eduardo/Chris see this, it's of course not just the xy plane, but the full 3 dimensions; hence the Bloch sphere representation)